tests/unit/test_utilities.py

```diff
@@ -358,3 +361,53 @@ def test_make_array_two_dimensional(self):
 
         self.assertEqual(len(output_array.shape), 2)
         np.testing.assert_array_equal(output_array, expected_output)
+
+
+class TestTransposeIfXdimLessThanYdim(TestCase):
+
+    def test_wider_than_tall(self):
+        """Test case where x dim <  y dim and should transpose."""
+        input_array = np.ma.array([[1, 2, 3], [4, 5, 6]])
+        expected_output = np.ma.array([[1, 4], [2, 5], [3, 6]])
```

Comment:
Just noting, related to the other comment, I don't think the input to this function is `np.ma.MaskedArray` - I think it's just a plain old `np.ndarray`. (This is true for all the inputs/outputs of this tests in this test class)

Reply:
I'm going to resolve this comment based on the proposed implementation in [this comment](https://github.com/nasa/harmony-swath-projector/pull/19/files#r1816724965).